pkg/supportbundle/supportbundle.go

```diff
@@ -189,11 +185,11 @@ func CreateSupportBundleDependencies(app apptypes.AppType, sequence int64, opts
 
 	supportBundleObj := types.SupportBundle{
 		AppID:      app.GetID(),
-		URI:        GetSpecURI(app.GetSlug()),
 		RedactURIs: redactURIs,
 		Progress: types.SupportBundleProgress{
 			CollectorCount: len(supportBundle.Spec.Collectors),
 		},
+		BundleSpec: supportBundle,
```

Comment:
We should change this back to the `URI` now that we are also preserving the merged `kotsadm-<app-slug>-supportbundle` spec?  It seems to cause the redactions to be omitted because `RedactURIs` are not used when [this code path](https://github.com/replicatedhq/kots/blob/main/pkg/supportbundle/execute.go#L191-L196) is taken.

Reply:
That's a good catch. I have rollback the change to use URI, until we change way of how RedactURIs to use same way like use tags to discover, let's keep it for now